ova/Config_files/ossec.conf

```diff
@@ -0,0 +1,18 @@
+<ossec_config>
```

Comment:
Can you change the indentation of this XML file? I mean, when a new block starts, increase the indentation to improve the readability of this file.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/06b971366820c9d9cbf359ba4c3b01c427772310